writing-apps/our-first-app/metadata.md

```diff
@@ -112,6 +112,10 @@ Your app must have a `release` tag for every version you wish to publish in AppC
         <li>An annoying behavior no longer occurs</li>
       </ul>
     </description>
+    <issues>
```

Comment:
I think we should update the description paragraph here as well. Currently it says to "avoid technical language", but issue tags should use the same title as the linked issue report according to the spec. So perhaps we should clarify that only the `description` tag should contain a non-technical description of changes and the `issues` tag should include any and all issues resolved in this release.

Reply:
How about reducing the size of this page in favor of linking to the _Publishing Updates_?